app/client/src/features/landing/Landing.tsx

```diff
@@ -56,6 +57,7 @@ export default function Landing() {
     const mdBreakpoint = useMediaQuery(theme.breakpoints.down('md'));
     const classes = useStyles();
     const router = useRouter();
+    const [isMobile] = React.useState(window.innerWidth < 640);
```

Comment:
This state doesn't update when the screen is resized, we want it to be reactive not just do a check when it's first loaded. You could add an effect to make it update when the page is resized, but I think it's a lot easier to just remove that isMobile state and use the existing media query there to do the check for rendering.